scripts/eqtl_hail_batch/conditional_analysis.py

```diff
@@ -375,6 +372,8 @@ def merge_significant_snps_dfs(*df_list):
     Merge list of list of sig_snps dataframes
     """
 
+    from multipy.fdr import qvalue 
```

Comment:
Maybe just add a comment here, something like:

```suggestion
    # This PythonJob is run in the multipy container, 
    # do the import here so it's not run in the driver container
    from multipy.fdr import qvalue 
```